.github/scripts/test-info.mjs

```diff
@@ -86,95 +99,136 @@ const components = {
             'AzureBlobStorageAccount',
             'AzureBlobStorageQueue',
         ],
-        sourcePkg: 'azure/storagequeues',
+        sourcePkg: [
+            'github.com/dapr/components-contrib/bindings/azure/storagequeues',
+        ],
     },
     'bindings.cron': {
         conformance: true,
         certification: true,
+        sourcePkg: ['github.com/dapr/components-contrib/bindings/cron'],
```

Comment:
Can we make this optional, or auto-generated, when the component uses only "standard" paths? This will make it easier to maintain.

The GenerateMatrix function below should be able to generate `github.com/dapr/components-contrib/bindings/cron` from `bindings.cron` without the user needing to type this.

Reply:
Please see the comment below